src/lib/components/search/display/images/Image.svelte

```diff
@@ -27,7 +27,7 @@
 		class:ring-4={selected}
 		class:ring-hearchco-primary={selected}
 		class:dark:ring-hearchco-secondary={selected}
-		class="w-full h-full overflow-hidden hover:scale-110 relative z-0 hover:z-50 hover:ring-4 hover:ring-gray-300 hover:dark:ring-gray-100 transform delay-150 duration-300 ease-in-out rounded-lg shadow-sm dark:shadow-none"
+		class="w-full h-full overflow-hidden hover:scale-110 relative z-0 hover:z-50 hover:ring-4 hover:ring-gray-300 hover:dark:ring-gray-100 transform delay-75 duration-300 ease-in-out rounded-lg shadow-sm dark:shadow-none"
```

Comment:
Delay change here might hinder some people who are not that fast with their mouse.
I've purposely added the highest delay recommended by Tailwind default classes as it is the only one which does not brokenly trigger the hover animation "half-way" while going over an image with reasonably slow speed.

Reply:
I don't see the problem you're describing with the value I set, which is currently deployed to production.

Reply:
I encountered this problem on Brave, even with your value set. So it will happen either way. It's better to not feel sluggish.

Reply:
Nice find, guess we'll have to find a different solution to this in the future then.